From 1f658eddc2a7b7b9c03aad9db9cdab0d49d71fa8 Mon Sep 17 00:00:00 2001
From: Vladimir Oltean <olteanv@gmail.com>
Date: Thu, 18 Oct 2018 00:44:02 +0300
Subject: [PATCH 20/26] extract_utils: avoid doing stuff on files that don't
 exist

* If an apk/jar doesn't exist, the script would still try to deodex it.
* If an xml doesn't exist, the script would still try to "fix" it.
* Take it easier, man, it's not your fault.

Change-Id: I3061fb48b403da5121e3c17dd9ecdb6cd148bf97
Signed-off-by: Vladimir Oltean <olteanv@gmail.com>
---
 build/tools/extract_utils.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/build/tools/extract_utils.sh b/build/tools/extract_utils.sh
index 9a43fc0e..23552037 100644
--- a/build/tools/extract_utils.sh
+++ b/build/tools/extract_utils.sh
@@ -1064,6 +1064,7 @@ function extract() {
 
             if [ "${FOUND}" = false ]; then
                 printf '    !! file not found in source\n'
+                continue
             fi
         fi
 
-- 
2.17.1

